parsec/test_utils/organization.py

```diff
@@ -116,16 +111,23 @@ async def initialize_test_organization(
                 # Add additional random users
                 if additional_users_number > 0:
                     await _add_random_users(
-                        alice_cmds,
-                        alice_device,
-                        config_dir,
-                        password,
-                        force,
-                        alice_core,
-                        bob_core,
-                        alice_ws_id,
-                        bob_ws_id,
-                        additional_users_number,
+                        cmds=alice_cmds,
+                        author=alice_device,
+                        alice_core=alice_core,
+                        bob_core=bob_core,
+                        alice_ws_id=alice_ws_id,
+                        bob_ws_id=bob_ws_id,
+                        additional_users_number=additional_users_number,
+                    )
+                # Add additional random device for alice
+                if additional_devices_number > 0:
```

Comment:
we can remove the if check and always call `_add_random_device` given it does nothing but a loop on `range(additional_devices_number )`